src/store.js

```diff
@@ -195,6 +195,14 @@ export class Store {
     return this._helper.lastResults.processingTimeMS;
   }
 
+  setMaxValuesPerFacet(limit) {
```

Comment:
All other options are using an ES6 setter syntax.
Can we do the same here for consistency?

Reply:
I don't think so, by definition a setter is used to modify a property of a class, this is not the case because we are updating the inner state of the helper to query with the correct `maxValuesPerFacet` parameter.

Reply:
Like you did for:

```js  
addFacetRefinement(attribute, value) {
    if (this._helper.state.isConjunctiveFacet(attribute)) {
      this._helper.addFacetRefinement(attribute, value);
    } else if (this._helper.state.isDisjunctiveFacet(attribute)) {
      this._helper.addDisjunctiveFacetRefinement(attribute, value);
    } else if (this._helper.state.isHierarchicalFacet(attribute)) {
      this._helper.addHierarchicalFacetRefinement(attribute, value);
    }
  }
```

Reply:
This is done on purpose everywhere in order to have Vue being able to observe the properties.
The Store acts as a proxy to the helper to circumvent some shortcomings of the helper.

Users can then `bind` values to that property.